content/updates/2020-10/5-New-filters-for-rules-conditions.md

```diff
@@ -9,15 +9,15 @@ pim_announcement_audience:
 ::: meta-data type="New" features="Automation, Productivity" available="early October" link-to-doc="../articles/manage-your-rules.html#available-conditions"
 :::
 
-We added new filters in the rules engine to help you better define your conditions. You can now filter on:
+New filters are available in the rules engine to help you better define your conditions. You can now filter on:
 - the entity type: to target product models or products and variant products,
 - family variants: to target products belonging to one or several family variants, or products with one or no family variant,
 - parents: to target specific product parents or products without parents
 - identifiers: to target specific products based on their product identifiers.
 
 ![The new filters in action](../img/new-filters-for-rules-conditions.png)
 
-Condition filters enable you to narrow the product selection to apply the rules on, so the more precise your rule conditions are, the faster your rule will be run. So don't hesitate to use those new filters.
+Condition filters enable you to narrow down the product selection for a given rule. The more precise your rule conditions, the faster it will be run. So don't hesitate to use those new filters.
```

Comment:
I feel "for a given rule" does not exactly convey the right meaning. Here, we want to say that if you select fewer products with your rule conditions (so your product selection is smaller), the rule will apply to fewer products. So it will run faster.

Reply:
I agree with Nolwenn. 
```suggestion
Condition filters enable you to narrow down the targeted products for a rule. The more precise your rule conditions, the faster it will be run. So don't hesitate to use those new filters.
```

Reply:
@camillebourjade what do you think of this new suggestion? ⬆️

Reply:
Ok that works for me!